samples/ResponseCachingSample/ResponseCachingSample.csproj

```diff
@@ -16,4 +16,7 @@
     <PackageReference Include="Microsoft.Extensions.Caching.Memory" Version="$(AspNetCoreVersion)" />
   </ItemGroup>
 
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net461' ">
```

Comment:
We are inconsistent here and from what I see in other repos we use net461 more often.